.env.sample

```diff
@@ -1,2 +1,3 @@
 # Generated at https://my.slack.com/services/new/bot
 # API_TOKEN =
+# API_TOKEN_DEV =
```

Comment:
I guess we should update the README to indicate where should we paste our token?